### PR TITLE
Add very minimal linting

### PR DESCRIPTION
**Pull request by itamarst**
Doesn't lint `eventlet/green/` and has a very restricted set of lints, but better than nothing.

### CODE REVIEW

File: eventlet/backdoor.py [hunk 1]
```diff
@@ -2,8 +2,6 @@
 import errno
 import socket
 import sys
-import errno
```
Comment (4383) [inline eventlet/backdoor.py:5]: errno is used many times in this module, we can't remove it. For an example, please, see line 37.

File: eventlet/wsgi.py [hunk 2]
```diff
@@ -474,7 +474,7 @@ def handle_one_request(self):
                 return
 
             if transfer_encoding is not None:
-                if reject_bad_requests:
+                if self.reject_bad_requests:
```
Comment (4383) [inline eventlet/wsgi.py:477]: good catch

File: eventlet/zipkin/api.py [hunk 3]
```diff
@@ -155,10 +155,11 @@ def build_span(name, trace_id, span_id, parent_id,
 
     @staticmethod
     def build_annotation(value, endpoint=None):
-        if isinstance(value, unicode):
+        if isinstance(value, str):
             value = value.encode('utf-8')
+        assert isinstance(value, bytes)
```
Comment (4383) [inline eventlet/zipkin/api.py:160]: please avoid using `assert` in runtime code. Could be replaced by something like:

```python
if not isinstance(value, bytes):
    raise RuntimeError("Only bytes are accepted...")
```

Comment (4383) [reply eventlet/zipkin/api.py:160]: Also, why adding this check?
`ttypes.Annotation` expect a string: https://github.com/eventlet/eventlet/blob/master/eventlet/zipkin/_thrift/zipkinCore/ttypes.py#L143

File: eventlet/zipkin/api.py [hunk 4]
```diff
@@ -155,10 +155,11 @@ def build_span(name, trace_id, span_id, parent_id,
...
         return ttypes.Annotation(time.time() * 1000 * 1000,
-                                 str(value), endpoint)
```
Comment (4383) [inline eventlet/zipkin/api.py:161]: I don't think we should remove that cast.
ttypes.Annotation expect a string: https://github.com/eventlet/eventlet/blob/master/eventlet/zipkin/_thrift/zipkinCore/ttypes.py#L143 and your previous check want to force for type bytes.

File: tests/greenpool_test.py [hunk 5]
```diff
@@ -7,11 +7,6 @@
 import tests
 
 
-def passthru(a):
```
Comment (4383) [inline tests/greenpool_test.py:10]: this function is used below, see line 29.

File: tests/ssl_test.py [hunk 6]
```diff
@@ -1,6 +1,4 @@
-import contextlib
 import random
-import socket
```
Comment (4383) [inline tests/ssl_test.py:3]: As `eventlet.green.socket` is imported below I think it is safe to remove this one.

**Review by 4383 — CHANGES_REQUESTED** [review]
Thanks for this patch, I think some parts needs changes, see my inline comments.

File: eventlet/backdoor.py (hunk 1 above)
Comment (itamarst) [reply eventlet/backdoor.py:5]: It's still imported, see line 2 right above it. The issue was it was being imported twice.

File: eventlet/zipkin/api.py (hunk 3 above)
Comment (itamarst) [reply eventlet/zipkin/api.py:160]: That's UTF-8 encoded string, i.e. bytes. AFAICT this code was never ported to Python 3, so when it says 'str()` in the original code it actually meant `bytes`. Notice I'm deleting a usage of `unicode()`... AFAICT from other uses of the code it expects bytes.

This is, to be fair, a guess though. This code is probably completely busted honestly given it wasn't ported to Python 3.

File: eventlet/zipkin/api.py (hunk 4 above)
Comment (itamarst) [reply eventlet/zipkin/api.py:161]: See above.

File: tests/greenpool_test.py (hunk 5 above)
Comment (itamarst) [reply tests/greenpool_test.py:10]: There were two `passthru` functions, there's another one defined later on.

File: eventlet/zipkin/api.py (hunk 3 above)
Comment (4383) [reply eventlet/zipkin/api.py:160]: ok thanks for your explanation.

File: eventlet/backdoor.py (hunk 1 above)
Comment (4383) [reply eventlet/backdoor.py:5]: Oh sorry, I missed the previous import.

File: eventlet/zipkin/api.py (hunk 4 above)
Comment (4383) [reply eventlet/zipkin/api.py:161]: thanks

File: tests/greenpool_test.py (hunk 5 above)
Comment (4383) [reply tests/greenpool_test.py:10]: lol :)

As the other is defined later, I suppose that this is this other one which is loaded on memory at runtime.

**Review by 4383 — APPROVED** [review]

**Pull request merged**